dotnet/src/SemanticKernel.UnitTests/KernelTests.cs

```diff
@@ -125,6 +126,34 @@ public void ItImportsSkillsNotCaseSensitive()
         Assert.True(skill.ContainsKey("GETANYVALUE"));
     }
 
+    [Theory]
+    [InlineData(null, "Assistant is a large language model.")]
```

Comment:
Where did this `Assistant is a large language model.` message come from?

Reply:
https://github.com/microsoft/semantic-kernel/blob/c8a7c6d134fff68e59dd8cc191e9a2fdc1d0937a/dotnet/src/SemanticKernel.Abstractions/AI/TextCompletion/CompleteRequestSettings.cs#L61